docker/entrypoint.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+# Define the start command
+START_COMMAND="teosd"
+
+# Set the API bind address
+if [[ ! -z ${API_BIND} ]]; then
+    START_COMMAND="$START_COMMAND --api-bind $API_BIND"
```

Comment:
None of the modifiers are actually correct for `teosd` for `rust-teos`. Go over `teosd --help` to see the proper ones.

e.g:

`--api-bind` -> `--apibind`

Reply:
Thanks! this command help a lot `teosd --help`. I will add new ones and remove old ones from `entrypoint.sh`